eslzArm/eslz-portal.json

```
@@ -2032,6 +2032,298 @@
+                            "visible":"[and(not(equals(steps('connectivity').enableHub, 'No')), not(equals(steps('connectivity').enableHub, 'nva')), not(equals(steps('connectivity').enableHub, 'vhub')))]",
+                            "toolTip":"Preference used in selecting best path when the virtual hub learns multiple paths to the same destination route-prefix.<a href=\"https://learn.microsoft.com/en-us/azure/virtual-wan/about-virtual-hub-routing-preference\">Virtual hub routing preference</a>.",
+                            "constraints":{ 
+                                "allowedValues":[
+                                    {
+                                        "label":"ExpressRoute",
+                                        "description":"ExpressRoute is the preferred path",
+                                        "value":"ExpressRoute"
+                                    },
+                                    {
+                                        "label":"VPN",
+                                        "description":"VPN is the preferred path",
+                                        "value":"VpnGateway"
+                                    },
+                                    {
+                                        "label":"AS Path",
+                                        "description":"AS Path is the preferred path",
+                                        "value":"ASPath"
+                                    }
+                                ]
+                            }
+                        },
+                        {
+                            "name": "vWANHubCapacity",
+                            "type": "Microsoft.Common.DropDown",
+                            "label": "Virtual Hub Capacity",
+                            "defaultValue": "",
+                            "multiselect": false,
+                            "selectAll": false,
+                            "filter": false,
+                            "multiLine": true,
+                            "visible":"[and(not(equals(steps('connectivity').enableHub, 'No')), not(equals(steps('connectivity').enableHub, 'nva')), not(equals(steps('connectivity').enableHub, 'vhub')))]",
+                            "toolTip": "Routing infrastructure units determine the minimum throughput of the Virtual WAN hub router and the number of Virtual Machines that can be deployed in Virtual Networks connected to the Virtual WAN hub. Two routing infrastructure units are included at no extra cost with a deployment of a hub.<a href=\"https://learn.microsoft.com/en-us/azure/virtual-wan/hub-settings#capacity\">Virtual Hub Capacity</a>.",
+                            "constraints": {
+                                "allowedValues": [
+                                    {
+                                        "label": "2",
+                                        "description": "2 Router Infrastructure Units, 3 Gbps Router, Supports 2000 VMs",
```

```suggestion
                                        "description": "2 Routing Infrastructure Units, 3 Gbps Aggregate Throughput, Supports 2000 VMs",
```

Can we change all of these to match the table column names here: https://learn.microsoft.com/en-us/azure/virtual-wan/hub-settings#routing-infrastructure-unit-table